### ISSUE TITLE
Training procedure

### ISSUE THREAD

**Issue by irwinherrmann** [d-li14/mobilenetv3.pytorch#7]
Specifically, what optimization algorithm, learning rate schedule, and batch size did you use?

**Comment by d-li14** [comment]
the same as https://github.com/d-li14/mobilenetv2.pytorch

**Comment by irwinherrmann** [comment]
Thanks! Btw, it might be useful to put this in the README.

Also, how many epochs did it take to converge?
